resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolver.java

```diff
@@ -536,17 +558,31 @@ private InetSocketAddress nextNameServerAddress() {
             Promise<AddressedEnvelope<? extends DnsResponse, InetSocketAddress>> promise) {
 
         return query0(checkNotNull(nameServerAddr, "nameServerAddr"),
-                      checkNotNull(question, "question"),
-                      checkNotNull(promise, "promise"));
+                checkNotNull(question, "question"), null, checkNotNull(promise, "promise"));
```

Comment:
I mean javax.annotation.Nullable. It can be used as a hint for API users... But it seems that Netty don't use it...